README.md

```diff
@@ -28,19 +28,23 @@ Find the full documentation [here](https://paritytech.github.io/asset-transfer-a
 
 The below chart is focusing on what directions are supported for constructing asset transfers and in what XCM version. The goal is to have everything in green checkmarks. Note that local transfers (intra-chain) are not visualized here.
 
+Note: Parachain To Parachain support is currently limited to XCM V2, with the exception of Parachain primary asset tx construction (e.g. MOVR, SDN, etc.)
+
 | Direction              | V2                 | V3                 |
 | ---------------------  | ------------------ | ------------------ |
 | System to Parachain    |         ✅         |      ✅            |
 | System to Relay        |         ✅         |      ✅            |
 | Relay to Parachain     |         ✅         |      ✅            |
 | Relay to System        |         ✅         |      ✅            |
-| Parachain to Parachain |         ❌         |      ❌            |
-| Parachain to Relay     |         ❌         |      ❌            |
+| Parachain to Parachain |         ✅         |      ❌            |
+| Parachain to Relay     |         ❌         |      ❎            |
```

Comment:
I think we need to put it on the para to para line, but i also think we can just do the Check mark for now, and write a note under explaining the situations we dont cover.

Reply:
Good eye. fixed!